src/indexer/index.ts

```diff
@@ -40,12 +40,12 @@ export const getApplicationGlobalState = async (
 		// get global state of application
 		const globalStateRes = await executeReq(lookupApplications(appId));
 		if (
-			globalStateRes?.allApps?.nodes.length &&
-			JSON.parse(globalStateRes.allApps.nodes[0].params).gs
+			globalStateRes?.allSigmaDaos?.nodes.length &&
```

Comment:
why??? We don't have sigma_daos table now. We are now directly using the app table now. Please setup sigmadao-indexer repo.

Reply:
Yes I have to update it
I forgot to add a TODO in the description.
I needed to test it and the indexer PR wasn’t merged till then thus reverted back 
Will update